scripts/postinstall.js

```diff
@@ -178,7 +174,10 @@ const downloadChromedriver = () => {
     .then(() => waitForFile(chromedriverUnzippedPath))
     .then(() => move(chromedriverUnzippedPath, chromedriverFinalPath))
     .then(() => fs.chmod(chromedriverFinalPath, '755'))
-    .then(() => waitForFile(chromedriverFinalPath));
+    .then(() => waitForFile(chromedriverFinalPath))
+    .catch((err) => {
```

Comment:
Might not need this since this catch will swallow errors and not allow them to "bubble up". Did you happen to find this useful? We can log then re-throw if so.

Reply:
Ah, yeah, leftover from the tests, sorry